runtime/src/main/java/org/corfudb/util/serializer/DynamicProtobufSerializer.java

```diff
@@ -151,6 +151,11 @@ private FileDescriptor getDescriptor(String name) throws DescriptorValidationExc
             return fileDescriptorMap.get(name);
         }
 
+        if (!fdProtoMap.containsKey(name)) {
+            log.error(this.toString());
```

Comment:
Can you add a log error message here, maybe log.error("cannot deserialize object {}", this, exception)